roles/installer/tasks/main.yml

```diff
@@ -23,6 +23,10 @@
     namespace: "{{ ansible_operator_meta.namespace }}"
   register: awx_web_deployment
 
+- name: Check for existing deployment for previous version
+  include_tasks: previous_version_check.yml
```

Comment:
check_previous_version.yml

Reply:
Thanks, I just address all these comments, thanks for the quick feedback!